### PR TITLE
Explain draft pull request

### PR DESCRIPTION
**Pull request by MaxenceGui**
Explain draft pull request

1. Open an [issue]((https://github.com/ai-cfia/dev-rel-docs/issues)
2. Name it "onboarding process of your name"
3. Use this template:

```
Maxence Guindon onboarding process

Tasks:

* [ ] Update time descriptions.

Acceptance Criteria:

The training should equip the developer with the skills necessary to work within the AI LAB's framework.
The training should provide accurate time estimates for each module.
```
4. Go back on training.md and click edit button on this file 
image
5. Edit the document (example add the time it took you to read value and ethics)
6. Click on commit changes
7. create a branch with this format: issue_number-onboarding-process-your-name (example 42-onboarding-process-happy-gilmor)
8. Click on propose change
9.  On the "Open a pull request" page:
10. Add a title
11. Select draft pull request instead of create pull request

### CODE REVIEW

**Comment by rngadam** [comment]
@MaxenceGui you asked for a review but have this set as draft?

**Comment by MaxenceGui** [comment]
Change it to ready to be reviewed

**Review by rngadam — APPROVED** [review]

**Review by rngadam — CHANGES_REQUESTED** [review]
@MaxenceGui cleanup the commits history by doing a rebase on the main branch (not a merge). 14 commits for a few lines changes is a bit much.

**Comment by MaxenceGui** [comment]
> @MaxenceGui cleanup the commits history by doing a rebase on the main branch (not a merge). 14 commits for a few lines changes is a bit much.

Would the simplest be:
image

**Comment by MaxenceGui** [comment]
@rngadam

Rebase is done. I keep only 3 commit with significant title.

File: TRAINING.md [hunk 1]
```diff
@@ -128,6 +154,12 @@
 * Nachet
   * [Introduction to blob
     store](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction)
+  * [Use Azure Storage Explorer
+    to create a blob](https://learn.microsoft.com/en-us/azure/storage/blobs/quickstart-storage-explorer)
+
+* Finessse
```
Comment (rngadam) [inline TRAINING.md:160]: Finesse

**Review by rngadam — APPROVED** [review]
fix typo first and then merge. notify @SamuelPelletierEvraire that the doc is updated.

**Pull request merged**